DigitalLearningSolutions.Web/ViewModels/SuperAdmin/Faqs/FaqsPageViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Web.Models.Enums;
+
+    public class FaqsPageViewModel
+    {
+        public FaqsPageViewModel(IEnumerable<SearchableFaqModel> faqs, SuperAdminSystemPage currentPage)
```

Comment:
Given I don't think the `CurrentPage` value will change can this be set directly on the property rather than passed into the model?